### ISSUE TITLE
shipments/create-command error

### ISSUE THREAD

**Issue by ZajdaBear** [allegro/allegro-api#10396]
Hi, there is an issue creating shipment.
command gats created, but it errors out down the line. Probably a carrier issue

< trace-id: 57eba5af970d5143
/shipment-management/shipments/create-commands/139dd382-56c8-4a99-9499-7ca73be277f0

response
````
{"commandId":"139dd382-56c8-4a99-9499-7ca73be277f0","status":"ERROR","errors":[{"code":"UNEXPECTED_ERROR","message":"Carrier returned unexpected error","details":"<html>\r\n<head><title>502 Bad Gateway</title></head>\r\n<body>\r\n<center><h1>502 Bad Gateway</h1>< [...]","path":"","userMessage":"Carrier returned unexpected error"}],"shipmentId":null}

**Comment by Lukasz-Zurek** [comment]
@ZajdaBear Yes, most probably there was a technical break on the logistic operator's side. Please let me know if the situation is ok right now and you can create a new shipment.

**Comment by SunkovyChlebiczek** [comment]
Hello,
we have the same issue with shipments from Allegro.pl, hu and sk and DPD orders.

< trace-id: 40e29aa7b3558576
/shipment-management/shipments/create-commands/709674d8-9405-42b2-a4d4-d623d556c623

response:
`{"commandId":"709674d8-9405-42b2-a4d4-d623d556c623","status":"ERROR","errors":[{"code":"UNKNOWN_ERROR","message":"Unexpected carrier error","details":"","path":"","userMessage":"Unexpected carrier error"}],"shipmentId":null}`

Is it the same issue?

**Comment by MartaNowaczyk** [comment]
@SunkovyChlebiczek Reply in thread #12610
